apps/web/pages/gallery/[galleryId]/edit.tsx

```diff
@@ -76,11 +80,29 @@ function EditGalleryPageInner({ queryRef }: EditGalleryPageInnerProps) {
   const handleBack = useGuardEditorUnsavedChanges(goBack, hasUnsavedChanges);
 
   const [isSaving, setIsSaving] = useState(false);
+
+  if (query.viewer?.__typename !== 'Viewer') {
+    throw new Error('Viewer does not exist');
+  }
+
+  const user = query.viewer.user;
+
   const handleSave = useCallback(async () => {
+    const userRoles = user?.roles ?? [];
+
     setIsSaving(true);
     await saveGallery();
     setIsSaving(false);
-  }, [saveGallery]);
+
+    // If the user is beta tester or admin, skip the modal
+    if (userRoles?.includes('BETA_TESTER') || userRoles?.includes('ADMIN')) {
+      return;
+    }
+
```

Comment:
oo would you be able to add a track('Show Beta Announcement Modal') event

Reply:
Updated!